<ui version="4.0" >
 <class>OptHistory</class>
 <widget class="QWidget" name="OptHistory" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptHistoryUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="ck_logMessages" >
     <property name="text" >
      <string>Enable logging messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_logFileTransfers" >
     <property name="text" >
      <string>Enable logging of file transfers</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel2" >
       <property name="text" >
        <string>Database backend to use:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_backend" >
       <item>
        <property name="text" >
         <string>SQLite 3</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Postgres</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>MySQL</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>ODBC</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_dbcon" >
     <property name="title" >
      <string>Database connection </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignJustify</set>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lb_hostname" >
        <property name="text" >
         <string>Hostname</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lb_username" >
        <property name="text" >
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lb_password" >
        <property name="text" >
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="lb_dbname" >
        <property name="text" >
         <string>Database name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLineEdit" name="le_username" />
      </item>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="le_hostname" />
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="le_password" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLineEdit" name="le_dbname" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="lb_port" >
        <property name="text" >
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2" >
       <widget class="QSpinBox" name="sb_port" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
